geopy/geocoders/googlev3.py

```diff
@@ -179,6 +180,11 @@ def geocode(
             .. versionchanged:: 1.14.0
                Now ``query`` is optional if ``components`` param is set.
 
+        :param str place_id: Address details are obtained using google
+                             place id.It is not used with the query or bounds
+                             parameter.
+                >>> g.geocode(place_id='ChIJOcfP0Iq2j4ARDrXUa7ZWs34')
```

Comment:
I suppose there should be an extra newline between the code sample and the `:param ...:` option. https://github.com/geopy/geopy/blob/master/CONTRIBUTING.md#building-docs -- this guide explains how to build the docs, could you please do that and ensure that it is rendered correctly?